src/collections/_documentation/accounts/teams.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Teams
+sidebar_order: 2
+---
+
+Teams are a workflow feature in Sentry that reflect the various departments and initiatives within a company's engineering organization. They are used to denote two things: access, and
+structure. By leveraging teams, an organization can introduce a sense of hierarchy, and route engineering groups directly to the projects that they work on.
```

Comment:
The benefit of "introduce a sense of hierarchy" is unclear to me. What if we changed these sentences to:

> Teams are a workflow feature in Sentry that are designed to reflect the teams within a company's engineering organization. They are used to denote two things: access, and
responsibility. By leveraging teams, an organization can scope a team's access in Sentry to the services they work on, triage issues to teams and team members, and create ownership rules that ensure the right teams and team members are alerted to problems as they emerge.

Reply:
Aside: I'm really excited for this section!